pytensor/tensor/nlinalg.py

```diff
@@ -582,26 +585,87 @@ def infer_shape(self, fgraph, node, shapes):
         else:
             return [s_shape]
 
+    def L_op(
+        self,
+        inputs: Sequence[Variable],
+        outputs: Sequence[Variable],
+        output_grads: Sequence[Variable],
+    ) -> list[Variable]:
+        (A,) = inputs
```

Comment:
We should add a link with attribution to the autograd implementation as a comment